src/main/java/de/tum/in/www1/artemis/web/rest/admin/AdminStandardizedCompetencyResource.java

```diff
@@ -107,12 +108,49 @@ public ResponseEntity<Void> deleteStandardizedCompetency(@PathVariable long comp
      */
     @PostMapping("standardized-competencies/knowledge-areas")
```

Comment:
not directly related to PR but `/standardized-competencies` could be moved to the RequestMapping Annotation of Resource class

Reply:
I actually tried that, but this causes the create standardized competencies endpoint to not work anymore (since the Spring Boot 3 upgrade) :/